### PR TITLE
Allow BSC, Polygon, Avalanche native and stable coins for on-ramp

### PR DESCRIPTION
**Pull request by wachunei**
**Description**

This PR allows buying AVAX in Avalanche and supported stablecoins in all networks (USDT, USDC, DAI, BUSD)

**~~⚠️ This PR includes changes from #3539~~ ℹ️ This PR superseeds #3539**

### CODE REVIEW

File: app/constants/on-ramp.js [hunk 1]
```diff
@@ -33,3 +33,15 @@ export const PAYMENT_CATEGORY = {
 	BANK_TRANSFER: 'Bank Transfer',
 	MULTIPLE: 'Multiple Options',
 };
+
+/**
+ * @enum {string}
+ */
+
+export const NETWORKS_CHAIN_ID = {
```
Comment (rickycodes) [inline app/constants/on-ramp.js:41]: similar stuff is defined for at least mainnet and kovan in `util/networks` as `NetworkList`. it looks like chain id there is `number` so maybe this small amount of duplicated id's is actually fine?

Comment (wachunei) [reply app/constants/on-ramp.js:41]: I decided to isolate these because the list can grow and new items could only be used in on-ramp flow.

**Comment by Cal-L** [comment]
Let's resolve this warning: 
Failed prop type: The prop `ticker` is marked as required in `PaymentMethodSelectorView`, but its value is `undefined`

**Comment by wachunei** [comment]
@Cal-L fixed in 05eaf1db79c53aa44f320f232c703defda0751cf

**Review by andrepimenta — APPROVED** [review]
LGTM!

**Pull request merged**